src/Runner.Worker/ActionManager.cs

```diff
@@ -53,30 +53,63 @@ public sealed class ActionManager : RunnerService, IActionManager
         public Dictionary<Guid, ContainerInfo> CachedActionContainers => _cachedActionContainers;
         public async Task<PrepareResult> PrepareActionsAsync(IExecutionContext executionContext, IEnumerable<Pipelines.JobStep> steps)
         {
+            // Assert inputs
```

Comment:
What happens for "./local"?

Reply:
local with containers is also interesting... wonder whether we support that today 🤔